tsconfig.node.json

```diff
@@ -2,7 +2,7 @@
 	"extends": "./tsconfig.base.json",
 	"compilerOptions": {
 		"lib": ["es2023"],
-		"module": "node16",
+		"module": "nodenext",
 		"moduleResolution": "node16",
```

Comment:
Could maybe also be nodenext, from the docs it seems to resolve to the same thing